app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FriendshipsController < ApplicationController
+  def create
+    friend = User.find_by(email: params[:email])
+    Friendship.create(user_id: current_user.id, friend_id: friend.id)
+    flash[:notice] = "#{friend.first_name} has been added to your friends list!"
+    redirect_to dashboard_path
```

Comment:
We should probably have a check for if it's not saved. Though, we should ideally never get here if the friendship isn't possible.